### PR TITLE
add draft content for universal search retirement

### PR DESCRIPTION
**Pull request by johngruen**
Adds draft retirement content for universal search...copy is still under review and will not be finalized until 11/21

### CODE REVIEW

**Comment by johngruen** [comment]
@clouserw ^

File: content-src/experiments/universal-search.yaml [hunk 1]
```diff
@@ -91,6 +91,68 @@ contributors:
 installation_count: 39943
 created: '2016-04-28T03:57:32.270681Z'
 modified: '2016-07-20T15:42:28.873321Z'
+completed: '2016-11-30T16:00:00.000000Z'
+graduation_report: >
+  <img src="/static/images/experiments/universal-search/experiments_experimentdetail/6/d/6d0ff50eff9223f0d0cc58c5aa5d0c81_image_1462827811_0995.jpg" alt="universal search preview" />
```
Comment (ghost) [inline content-src/experiments/universal-search.yaml:97]: Is this image going to exist when this goes out?  (I don't see it in this PR)  Same with the one below

File: content-src/experiments/universal-search.yaml [hunk 2]
```diff
@@ -91,6 +91,68 @@ contributors:
...
+  <p>In October 2016, we added a third result type, rich movie cards, which we
+  hypothesized would be shown less often but clicked more frequently than the
+  other types. A breakdown of clicks by result type showed that Wikipedia pages
+  were shown most and clicked least. Top-level domains were the most consistent
+  performer. Movie Cards had the least amount of test data, but outperformed
+  Wikipedia pages. In aggregate, our click-through-rate for search results
+  settled around 7% by the end of the experiment. In the graphs below, we have a
+  data gap from July through the first week of August where we upgraded Test
+  Pilot Telemetry.</p>
+  <p>Besides telemetry data, we also collected user sentiment via surveys. Users
```
Comment (ghost) [inline content-src/experiments/universal-search.yaml]: I think "telemetry" should be capitalized here, but willing to be told I'm wrong

Comment (johngruen) [reply content-src/experiments/universal-search.yaml]: @clouserw sure

File: frontend/src/app/containers/ExperimentPage.js [hunk 3]
```diff
@@ -304,36 +304,38 @@ export class ExperimentDetail extends React.Component {
...
-                          <td><a data-hook="bug-report-url" href={bug_report_url}>{bug_report_url}</a></td>
-                        </tr>
-
-                        <tr>
-                          <td data-l10n-id="discourse">Discourse</td>
-                          <td><a data-hook="discourse-url" href={discourse_url}>{discourse_url}</a></td>
-                        </tr>
-                      </tbody></table>
-                    </section>}
+                    {!graduated && <div>
```
Comment (ghost) [inline frontend/src/app/containers/ExperimentPage.js:325]: Where does "graduated" come from?  Set somewhere from the completed field in the yaml?

File: content-src/experiments/universal-search.yaml (hunk 1 above)
Comment (johngruen) [reply content-src/experiments/universal-search.yaml:97]: @clouserw they're copied over from the details images, so yes, unless we remove them which i don't think we would.

File: frontend/src/app/containers/ExperimentPage.js (hunk 3 above)
Comment (johngruen) [reply frontend/src/app/containers/ExperimentPage.js:325]: https://github.com/mozilla/testpilot/blob/beee166afc2125c0b7cdbd4af051b45709af897f/frontend/src/app/containers/ExperimentPage.js#L222

Comment (johngruen) [reply frontend/src/app/containers/ExperimentPage.js:325]: it's generated by checking the current time against the graduation date

File: content-src/experiments/universal-search.yaml [hunk 4]
```diff
@@ -91,6 +91,68 @@ contributors:
...
+  surprisingly high click-through rate for serving a mere three results types,
+  and user sentiment was quite positive. The prototype we built was not a
+  complete product experience, but it was enough to meet our learning goals, and
+  we will now retire the experiment. If we tried a similar experiment again,
+  we’d pursue better result relevancy and more diverse result types. As part of
+  this first experiment, we did not test alternate UX treatments to see if we
+  could affect interaction with results, or improve user acceptance of the
+  combined Awesome Bar for those users who preferred separate bars. These are
+  questions we may examine in future experiments.</p>
+  <div class="float-right"><strong>-Javaun Moradi, Product Manager<strong></div>
```
Comment (pdehaan) [inline content-src/experiments/universal-search.yaml]: Should be `</strong></div>` at end of line.

Comment (johngruen) [reply content-src/experiments/universal-search.yaml]: Thanks!

**Pull request merged**